scripts/check/surface.py

```
@@ -0,0 +1,303 @@
+    default ``MCB_`` prefix while keeping the FLEXT settings lifecycle.
+    """
+
+
+def _run_case(case: SurfaceCase) -> str | None:
+    env = os.environ.copy()
+    env["APPLY"] = "N"
+    env["QUICK"] = "1"
+    env["TERM"] = "dumb"
+
+    make_bin = shutil.which("make") or "make"
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)